pyani_plus/public_cli.py

```diff
@@ -870,6 +873,7 @@ def float_or_na(value: float | None) -> str:
         (run.sim_errors, f"{method}_sim_errors.tsv"),
         (run.cov_query, f"{method}_query_cov.tsv"),
         (run.hadamard, f"{method}_hadamard.tsv"),
+        (run.tani, f"{method}_tANI.tsv"),
```

Comment:
See the refactoring done for the plots to avoid loading all the matrices at once before exporting them one-by-one. Could do the same here (although memory not such a problem as not making heavy graphs)?